### PR TITLE
SAT-409 Show current email 

### PR DESCRIPTION
**Pull request by mbortsov**

### CODE REVIEW

File: lib/translation/en.dart [hunk 1]
```diff
@@ -62,8 +62,9 @@ const Map<String, String> en = {
   'txt_claim_rewards': 'Claim rewards',
   'txt_congrats': 'Congratulations',
   'txt_congrats_msg': 'You have successfully claimed your rewards.',
-  'txt_verification_text':
-      'Please enter the verification code we’ve sent to your email address or use the link from your inbox',
+  'txt_verification_text_1':
+      'Please enter the verification code we’ve sent to ',
+  'txt_verification_text_2': ' or use the link from your inbox',
```
Comment (MichaelBoostyLabs) [inline lib/translation/en.dart:67]: use string with parameters instead 2 parts

File: lib/ui/page_widget/email_verification_page.dart [hunk 2]
```diff
@@ -34,12 +34,36 @@ class EmailVerificationPage extends GetView<EmailVerificationController> {
                   SizedBox(
                     height: 6,
                   ),
-                  Text(
-                    "txt_verification_text".tr,
-                    style: TextStyle(
-                        color: SatorioColor.textBlack,
-                        fontSize: 15.0,
-                        fontWeight: FontWeight.w400),
+                  Row(
```
Comment (MichaelBoostyLabs) [inline lib/ui/page_widget/email_verification_page.dart:37]: use simple Text with formatted string

**Review by MichaelBoostyLabs — CHANGES_REQUESTED** [review]

File: lib/translation/en.dart (hunk 1 above)
Comment (mbortsov) [reply lib/translation/en.dart]: done

File: lib/ui/page_widget/email_verification_page.dart (hunk 2 above)
Comment (mbortsov) [reply lib/ui/page_widget/email_verification_page.dart:38]: done

**Review by MichaelBoostyLabs — APPROVED** [review]

**Pull request merged**